dream_layer_backend/txt2img_workflow.py

```
@@ -1,7 +1,9 @@
 import json
 import random
 import os
+import logging
 import json
```

Ruff (0.12.2)

Redefinition of unused `json` from line 1

Remove definition: `json`

(F811)